src/isar_exr/config/maps/exr_default.json

```diff
@@ -1,41 +1,41 @@
   {
-    "name": "klab_vgii",
+    "name": "simulator",
     "map_from": {
       "name": "exr_default_robot",
       "reference_positions": {
         "positions": [
-        {
-          "x": -4.45,
-          "y": 0.66,
-          "z": -0.02,
-          "frame": {
-            "name": "robot"
-          }
-        },
-        {
-          "x": -6.64,
-          "y": 9.88,
-          "z": 0.87,
-          "frame": {
-            "name": "robot"
-          }
-        },
-        {
-          "x": 1.97,
-          "y": -0.89,
-          "z": 0.03,
-          "frame": {
-            "name": "robot"
-         }
-        },
-        {
-          "x": -9.58,
-          "y": -1.75,
-          "z": -0.18,
-          "frame": {
-            "name": "robot"
+          {
+            "x": 0,
+            "y": 0,
+            "z": 0,
+            "frame": {
+              "name": "robot"
+            }
+          },
+          {
+            "x": -6.64,
```

Comment:
These values could maybe be less random by using full integers but I guess it doesn't really matter

Reply:
Sure, changed them to whole integers.